zulipterminal/core.py

```diff
@@ -274,6 +276,12 @@ def show_msg_info(
         )
         self.show_pop_up(msg_info_view, "area:msg")
 
+    def show_file_upload_popup(self, write_box: WriteBox) -> None:
+        file_upload_view = FileUploadView(
+            self, write_box, "Upload File(Enter the location)"
```

Comment:
There's no space before the parenthesis here, but let's keep the title simple and drop those and the text inside. The action should be clear from the internals of the popup?